### PR TITLE
add additional bootstrap attributes in html sanitization

### PR DESCRIPTION
**Pull request by TristanB17**
# PR Checklist

Please check if your PR fulfills the following requirements
- [x] The title follows our [guidelines](https://github.com/ideacrew/enroll/blob/trunk/CONTRIBUTING.md#commit
)
- [x] Tests for the changes have been added (for bugfixes / features)

# PR Type
What kind of change does this PR introduce?

- [x] Bugfix
- [ ] Feature (requires Feature flag)
- [ ] Data fix or migration (inert code, no impact until run)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [ ] Dependency updates (e.g., add a new gem or update version)

# What is the ticket # detailing the issue?

Ticket: [187106404](https://www.pivotaltracker.com/story/show/187106404)

# A brief description of the changes

Current behavior: Tooltip on `/benefit_sponsors/profiles/employers/employer_profiles/:id?tab=home` not static nor styled correctly due to `data-toggle` attribute being removed during sanitization

New behavior: Tooltip now displays statically (not on hover) with correct styling

# Feature Flag

For all new feature development, a feature flag is required to control the exposure of the feature to our end users. A feature flag needs a corresponding environment variable that is used to initialize the state of the flag. Please share the name of the environment variable below that would enable/disable the feature and which client(s) it applies to.

Variable name:

- [ ] DC
- [ ] ME

# Additional Context
Include any additional context that may be relevant to the peer review process.

# AppScan CodeSweep Failure
In the event of a failed check on the AppScan CodeSweep step of our GitHub Actions workflow, please review the False Positive protocol outlined here: appscan_codesweep/CODESWEEP_FALSE_POSITIVES_README.MD

Add all required notes to this section if the failure is a suspected false positive.

### CODE REVIEW

File: lib/html_scrubber_util.rb [hunk 1]
```diff
@@ -10,7 +10,11 @@ def sanitize_html(value)
     ActionController::Base.helpers.sanitize(
       value,
       tags: Loofah::HTML5::SafeList::ACCEPTABLE_ELEMENTS.dup.delete("select").add("style"),
-      attributes: Loofah::HTML5::SafeList::ACCEPTABLE_ATTRIBUTES.dup.add("style")
+      attributes: Loofah::HTML5::SafeList::ACCEPTABLE_ATTRIBUTES.dup
+        .add("style")
+        .add("data-toggle")
+        .add("data-slide-to")
+        .add("data-target")
```
Comment (ATBull81) [inline lib/html_scrubber_util.rb:17]: Small request, could we change this to use `merge` similar to the update to pdf_scrubber_util? https://github.com/ideacrew/enroll/blob/d1febb2bc9a00a73862701605bf5dc95a67f8acc/lib/pdf_scrubber_util.rb#L12

Comment (TristanB17) [reply lib/html_scrubber_util.rb]: Yessir, file is prettified!

File: lib/html_scrubber_util.rb [hunk 2]
```diff
@@ -9,8 +9,15 @@ module HtmlScrubberUtil
...
       value,
-      tags: Loofah::HTML5::SafeList::ACCEPTABLE_ELEMENTS.dup.delete("select").add("style"),
-      attributes: Loofah::HTML5::SafeList::ACCEPTABLE_ATTRIBUTES.dup.add("style")
+      tags: Loofah::HTML5::SafeList::ACCEPTABLE_ELEMENTS.dup.delete('select').add('style'),
+      attributes: Loofah::HTML5::SafeList::ACCEPTABLE_ATTRIBUTES.dup.merge(
+        [
+          'style',
+          'data-toggle',
+          'data-slide-to',
+          'data-target'
```
Comment (ATBull81) [inline lib/html_scrubber_util.rb:18]: Could we add specs for these in https://github.com/ideacrew/enroll/blob/trunk/spec/lib/html_scrubber_util_spec.rb?

Comment (TristanB17) [reply lib/html_scrubber_util.rb:18]: Added!

**Review by ATBull81 — DISMISSED** [review]

File: spec/lib/html_scrubber_util_spec.rb [hunk 3]
```diff
@@ -42,11 +42,38 @@ class TestingScrubber
   end
 
   context "given a style attribute" do
-    let(:source_html) { "<div style=\"somewhere\"></div>" }
+    let(:source_html) { "<style>Test</style>" }
```
Comment (mdkaraman) [inline spec/lib/html_scrubber_util_spec.rb:45]: Why the change here?  This contradicts the context's description: "given a style attribute".  We should keep the descriptions aligned with what's being tested.

Comment (ATBull81) [reply spec/lib/html_scrubber_util_spec.rb:45]: Had talked off GitHub about this with Tristan; The previous PR to add this test accidentally duplicated the test (https://github.com/ideacrew/enroll/blob/trunk/spec/lib/html_scrubber_util_spec.rb#L35-L51) when one of them should'vebeen testing the style tag vs the style attribute. Maybe the only other thing is to change the context of this test to say "given a style tag"

Comment (TristanB17) [reply spec/lib/html_scrubber_util_spec.rb:45]: @ATBull81 if I'm understanding correctly, we also want to include style tags? Going to update description

Comment (TristanB17) [reply spec/lib/html_scrubber_util_spec.rb:45]: Updated! Let me know if the new change works

**Review by ATBull81 — APPROVED** [review]

**Review by mdkaraman — APPROVED** [review]
Looks good!
